src/content/docs/infrastructure/host-integrations/infrastructure-integrations-sdk/specifications/integration-logging-recommendations.mdx

```diff
@@ -12,7 +12,7 @@ redirects:
   - /docs/integrations/integrations-sdk/file-specifications/integration-logging-recommendations
 ---
 
-[New Relic Infrastructure](/docs/infrastructure/new-relic-infrastructure/getting-started/introduction-new-relic-infrastructure) provides an SDK for creating an on-host integration. This document explains requirements and best practices for generating integration logs.
+[New Relic's infrastructure agent](/docs/infrastructure/new-relic-infrastructure/getting-started/introduction-new-relic-infrastructure) provides an SDK for creating an on-host integration. This document explains requirements and best practices for generating integration logs.
```

Comment:
i don't think it's the infrastructure agent that provides the SDK, maybe we just say 'New Relic provides an SDK for creating an on-host integration'?

Reply:
Good catch!

Reply:
Verified - it does exist. Doing some minor tweaking to the doc though.